src/main/java/io/cryostat/recordings/Recordings.java

```diff
@@ -558,10 +531,7 @@ public Uni<Response> createSnapshotV2(@RestPath URI connectUrl) throws Exception
                 .transform(
                         recording ->
                                 Response.status(Response.Status.CREATED)
-                                        .entity(
-                                                V2Response.json(
-                                                        Response.Status.CREATED,
-                                                        recordingHelper.toExternalForm(recording)))
+                                        .entity(recordingHelper.toExternalForm(recording))
```

Comment:
We want to remove the `V2Response` format, but doing this should also mean removing or renaming any endpoints that return it. So this `/api/v2` endpoint should either be removed, or it should be renamed to `/api/v3` if it will return this simplified JSON response format directly.